### PR TITLE
Export backups to a sub-folder

### PR DESCRIPTION
**Pull request by levigideon**
### First time contributor checklist

- [x] I have read [how to contribute](https://github.com/WhisperSystems/Signal-Android/blob/master/CONTRIBUTING.md) to this project
- [x] I have signed the [Contributor Licence Agreement](https://whispersystems.org/cla/)

### Contributor checklist

- [x] I am following the [Code Style Guidelines](https://github.com/WhisperSystems/Signal-Android/wiki/Code-Style-Guidelines)
- [x] I have tested my contribution on these devices:
 * Alcatel POP3, Android 6.0
 * Virtual Nexus 5, Android 7.1.1
- [x] My contribution is fully baked and ready to be merged as is
- [x] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)
- [x] I have made the choice whether I want the [BitHub reward](https://github.com/WhisperSystems/Signal-Android/wiki/BitHub-Rewards) or not by omitting or adding the word `FREEBIE` in the commit message of my first commit

----------

### Description
Modified the exporter functions to export to a 'SignalBackups' folder instead of the SD card root and modified the importer functions to import from the 'SignalBackups' folder if available.

Testing:
1. Made a backup before updating, confirmed at /SignalPlaintextBackup.xml
2. Installed build with fix
3. Exported again
4. Confirmed that the new file was at /SignalBackups/SignalPlaintextBackup.xml
5. Successfully imported file 
6. Installed build without fix
7. Removed /SignalBackups/SignalPlaintextBackup.xml file manually
8. Made a backup to /SignalPlaintextBackup.xml
9. Installed build with fix
10. Successfully imported old file

### CODE REVIEW

File: src/org/thoughtcrime/securesms/database/PlaintextBackupExporter.java [hunk 1]
```diff
@@ -13,11 +13,13 @@
 public class PlaintextBackupExporter {
 
   private static final String FILENAME = "SignalPlaintextBackup.xml";
+  private static final String BACKUPFOLDERNAME = "SignalBackups";
```
Comment (FeuRenard) [inline src/org/thoughtcrime/securesms/database/PlaintextBackupExporter.java]: align equal signs

File: src/org/thoughtcrime/securesms/database/PlaintextBackupExporter.java [hunk 2]
```diff
@@ -26,8 +28,27 @@ private static void verifyExternalStorageForPlaintextExport() throws NoExternalS
       throw new NoExternalStorageException();
   }
 
-  public static File getPlaintextExportFile() {
+  private static void moveOldPlaintextExportToDirectory(){
+    File oldBackup = getOldPlaintextExportFile();
+    File newBackup = getPlaintextExportFile();
+    File newDirectory = new File(Environment.getExternalStorageDirectory() + File.separator + BACKUPFOLDERNAME + File.separator);
```
Comment (FeuRenard) [inline src/org/thoughtcrime/securesms/database/PlaintextBackupExporter.java]: align equal signs

File: src/org/thoughtcrime/securesms/database/PlaintextBackupImporter.java [hunk 3]
```diff
@@ -36,10 +36,16 @@ private static void verifyExternalStorageForPlaintextImport() throws NoExternalS
   private static File getPlaintextExportFile() {
     File backup    = PlaintextBackupExporter.getPlaintextExportFile();
     File oldBackup = new File(Environment.getExternalStorageDirectory(), "TextSecurePlaintextBackup.xml");
+    File oldRenamedBackup = new File(Environment.getExternalStorageDirectory(), "SignalPlaintextBackup.xml");
```
Comment (FeuRenard) [inline src/org/thoughtcrime/securesms/database/PlaintextBackupImporter.java]: align equal signs

**Comment by nrizzio** [comment]
This pull request is obsolete and should be closed because the new backup system already saves backups in a subfolder.

**Pull request closed without merging**